.github/workflows/cd-i18next.yml

```
@@ -0,0 +1,69 @@
+name: Continuous Delivery - i18next
+on:
```

```suggestion
name: Continuous Delivery - i18next

on:
```

And check whitelines compared to other cd files.